docs/data/data-grid/column-groups/BasicGroupingDemo.js

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import { DataGridPro } from '@mui/x-data-grid-pro';
+
+const columns = [
+  { field: 'id', headerName: 'ID', width: 90 },
+  {
+    field: 'firstName',
+    headerName: 'First name',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'lastName',
+    headerName: 'Last name',
+    width: 150,
+    editable: true,
+  },
+  {
+    field: 'age',
+    headerName: 'Age',
+    type: 'number',
+    width: 110,
+    editable: true,
+  },
+];
+
+const rows = [
+  { id: 1, lastName: 'Snow', firstName: 'Jon', age: 35 },
+  { id: 2, lastName: 'Lannister', firstName: 'Cersei', age: 42 },
+  { id: 3, lastName: 'Lannister', firstName: 'Jaime', age: 45 },
+  { id: 4, lastName: 'Stark', firstName: 'Arya', age: 16 },
+  { id: 5, lastName: 'Targaryen', firstName: 'Daenerys', age: null },
+  { id: 6, lastName: 'Melisandre', firstName: null, age: 150 },
+  { id: 7, lastName: 'Clifford', firstName: 'Ferrara', age: 44 },
+  { id: 8, lastName: 'Frances', firstName: 'Rossini', age: 36 },
+  { id: 9, lastName: 'Roxie', firstName: 'Harvey', age: 65 },
+];
+
+export default function BasicGroupingDemo() {
+  return (
+    <div style={{ marginTop: '20px', height: 400, width: '100%' }}>
+      <DataGridPro
+        sx={{
+          '& .groupingHeader': {
+            border: 'none',
+            borderLeft: 'solid gray 1px',
+          },
+          '& .groupingHeader.groupingHeader-withName': {
+            borderBottom: 'solid black 1px',
+          },
+        }}
+        rows={rows}
+        columns={columns}
+        pageSize={5}
+        rowsPerPageOptions={[5]}
+        checkboxSelection
+        disableSelectionOnClick
+        columnGroupingModel={[
+          {
+            headerName: 'internal',
+            children: [{ field: 'id' }],
+          },
+          {
+            headerName: 'perso',
```

Comment:
Is it possible to allow navigating column groups with keyboard?
I'm not sure it's super useful, but if feels unnatural not being able to go up from the original column header cells.
AG Grid allows to navigate column group cells https://www.ag-grid.com/javascript-data-grid/column-groups/

Reply:
For now, I did not implement it, mostly because there is no action to do in the column headers, but when it will be possible to google the view yes we should be able to add keyboard navigation